src/Database/Firebase.hs

```diff
@@ -32,22 +32,75 @@ query = Query Nothing Nothing Nothing Nothing
 key :: ToJSON a => a -> Key
 key = Key
 
-get :: (MonadIO m, MonadError e m, MonadReader r m, AsHttpError e, HasFirebase r, HasHttpCfg r, FromJSON a) => Location -> Maybe Query -> m a
+get :: (
+  MonadIO m, 
+  MonadError e m, 
+  MonadReader r m,
+  AsHttpError e,
+  HasFirebase r,
+  HasHttpCfg r,
+  FromJSON a) => 
+    Location -> 
```

Comment:
No, you usually wont' have a copy of the object you're trying to get, although you may know where it is.